news/2023/2023-02-17-community-contributors-elite-nominators-2022.md

```diff
@@ -0,0 +1,125 @@
+
+## osu!mania
+
+[![](/wiki/shared/news/2023-02-17-community-contributors-elite-nominators-2022/Maxus.jpg)](https://osu.ppy.sh/users/4335785)
+
+What isn't there to say about Maxus? If you were around for most of 2022, then you've probably heard his name at some point, as he was easily one of the most active and contributing members on the team at the time, and remains so. He's been around in the community for many years now whilst being under a different alias at some point, so seeing him achieve this amount of growth within only a year is absolutely remarkable.
+
+This involvement ultimately resulted in him attending to a whopping total of 125 nominations in 2022, which is not only incredible, but also record-breaking as this the most number of nominations done in a year by a singular nominator in osu!mania's history! Make no mistake — he did so without compromising quality, tackling a diverse range of mapping styles that left considerable impact in the community at large. He pulled this off all while involved in many other projects and the various teams, all but cementing his significance in the osu!mania community during the year.
+
+Maxus took massive strides to make mania a lot more active as a nominator in 2022, and we hope to see the same great strides from him in 2023 as well!
+
+[![](/wiki/shared/news/2023-02-17-community-contributors-elite-nominators-2022/gzdongsheng.jpg)](https://osu.ppy.sh/users/8660315)
+
+Being another Mania powerhouse coming into the scene in early 2022, gzdongsheng has been a prime example of quality for other nominators by nominating various high level maps, maps with huge spreads and lengthy drain times, and creations from different mappers across the community. Reaching over 103 nominations, he has proven himself to be an unstoppable force, opening the gates for much more diverse content to enter ranked, all backed up by an impressively consistent modding and nomination quality across the board to boot.
+
+He is one of the few nominators brave enough to push novel high level 4K maps prone to be controversial and that would otherwise have a difficult time getting into the ranked section. Despite the skill ceiling, he manages to work on them skillfully, showcasing his experience and expertise as a nominator. With the amount of dedication and effort that he put in addition to promoting a variety of keymodes, gzdongsheng has proven himself to be one of the top nominators that are more than deserving to be awarded the coveted title of Elite Nominator!
```

Comment:
```suggestion
He is one of the few nominators brave enough to push novel and high-level 4K maps prone to controversy that would otherwise have a difficult time getting into the Ranked section. Despite the skill ceiling, his ability to work on them skilfully showcases his experience and expertise as a nominator. With his amount of dedication and effort, in addition to promoting a variety of keymodes, gzdongsheng has proven himself to be one of the top nominators that are more than deserving to be awarded the coveted title of Elite Nominator!
```